cpp/log/cluster_state_controller-inl.h

```diff
@@ -446,11 +449,14 @@ void ClusterStateController<Logged>::DetermineElectionParticipation(
 template <class Logged>
 void ClusterStateController<Logged>::ClusterServingSTHUpdater() {
   while (true) {
+    LOG(INFO) << "ClusterServingSTHUpdater going again.";
     std::unique_lock<std::mutex> lock(mutex_);
     update_required_cv_.wait(lock, [this]() {
       return update_required_ || exiting_;
     });
+    LOG(INFO) << "ClusterServingSTHUpdater got ping.";
     if (exiting_) {
+      LOG(INFO) << "ClusterServingSTHUpdater thread returning.";
```

Comment:
They kinda slipped in there, but they might be useful again the in future so I've mostly cranked them down to `VLOG(1)`, a couple remain `LOG(INFO)` though.